### PR TITLE
[Silabs] : Cleanup and applying coding guideline on Silabs code

### PR DESCRIPTION
**Pull request by sharad-patil24**
This PR is the first PR in the series of PR's to cleanup and Update our code to follow coding style guideline provided for Silabs.

In this PR we focused on the necessary ( Must do ) things.

### CODE REVIEW

File: src/platform/silabs/SiWx917/wifi/wfx_host_events.h [hunk 1]
```diff
@@ -57,77 +57,77 @@
...
+#define WFX_SPI_NVIC_PRIORITY (5)
+#define WFX_GPIO_NVIC_PRIORITY (5)
+#define CB_VALUE ((DMADRV_Callback_t) 0)
 
 /* TIMER_TICKS_TO_WAIT Specifies the time, in ticks, that the calling task should
  * be held in the Blocked state to wait for the start command to be successfully
  * sent to the timer command queue.
  */
-#define TIMER_TICKS_TO_WAIT_0 pdMS_TO_TICKS(0)
+#define TIMER_TICKS_TO_WAIT_0 (pdMS_TO_TICKS(0))
```
Comment (mkardous-silabs) [inline src/platform/silabs/SiWx917/wifi/wfx_host_events.h:113]: not sure we need a define for 0

Comment (jmartinez-silabs) [reply src/platform/silabs/SiWx917/wifi/wfx_host_events.h:113]: The define doesn't bother me but i wonder why a wait of 0 is needed

File: src/platform/silabs/SiWx917/BLEManagerImpl.cpp [hunk 2]
```diff
@@ -493,12 +494,11 @@ CHIP_ERROR BLEManagerImpl::MapBLEError(int bleErr)
 
 void BLEManagerImpl::DriveBLEState(void)
 {
-
     ChipLogProgress(DeviceLayer, "DriveBLEState starting");
     CHIP_ERROR err = CHIP_NO_ERROR;
 
     // Check if BLE stack is initialized
-    // VerifyOrExit(mFlags.Has(Flags::kEFRBLEStackInitialized), /* */);
+    // VerifyOrExit(mFlags.Has(Flags::kEFRBLEStackInitialized),);
```
[the remark this answers is not in the document]
Comment (sharad-patil24) [reply src/platform/silabs/SiWx917/BLEManagerImpl.cpp:501]: I think we can change it in another PR.
I am not the implementer for this code and does not wants to change the logic.

File: src/platform/silabs/SiWx917/wifi/wfx_host_events.h (hunk 1 above)
Comment (sharad-patil24) [reply src/platform/silabs/SiWx917/wifi/wfx_host_events.h:113]: Yes it is odd , and I checked it , we are not using it in our code.
So removed this MACRO

**Review by jmartinez-silabs — APPROVED** [review]

**Review by mkardous-silabs — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]

**Pull request merged**